R/launcher-functions.R

```diff
@@ -128,11 +132,13 @@ launcherPlacementConstraint <- function(name,
 
 #' Define a Launcher Resource Limit
 #'
-#' Define a launcher resource limit, suitable for use with the
-#' `resourceLimits` argument to [launcherSubmitJob()].
+#' Define a launcher resource limit, suitable for use with the `resourceLimits`
+#' argument to [launcherSubmitJob()].
 #'
 #' @param type The resource limit type. Must be one of cpuCount, cpuFrequency,
-#'   cpuSet, cpuTime, memory, memorySwap.
+#'   cpuSet, cpuTime, memory, memorySwap. Different launcher plugs may support
```

Comment:
Typo: "plugs"

Reply:
Doh; thanks for catching that. Will fix up and merge; thanks for the review!